.github/ISSUE_TEMPLATE/Documentation.yml

```diff
@@ -0,0 +1,30 @@
+name: 📄 Documentation issue
+description: Found an issue in the documentation? You can use this one!
+title: "[DOCS]"
+labels: ["documentation"]
+
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A brief description of the question or issue, also include what you tried and what didn't work
+    validations:
+      required: true
+
+  - type: textarea
+    id: extrainfo
+    attributes:
+      label: Additional information
+      description: Is there anything else we should know about this issue?
+    validations:
+      required: false
+
+   - type: checkboxes
```

Comment:
There is an additional space before `- type: ` that causes this error:

> YAML syntax error: (): did not find expected '-' indicator while parsing a block collection at line 7 column 3.

Remove that space may fix this issue.